src/component/canvas/ScatterCanvas.jsx

```diff
@@ -0,0 +1,145 @@
+// @flow
+
+import React from 'react';
+import Canvas from './Canvas';
+import type {ChartScalar, ChartRow} from 'src/chartdata/ChartData';
+
+type DotProps = {
+    key: string,
+    x: number,
+    y: number,
+    dataX: ChartScalar,
+    dataY: ChartScalar,
+    row: ChartRow
+};
+
+/**
+ *
+ * @typedef Dot
+ * @type ReactElement
+ *
+ * @prop {number} x - The x position of the dot on the canvas
+ * @prop {number} y - The y position of the dot on the canvas
+ * @prop {ChartScalar} dataX - The `ChartScalar` value for `columnX` in this row.
+ * @prop {ChartScalar} dataY - The `ChartScalar` value for `columnY` in this row.
+ * @prop {ChartRow} row - The `ChartRow` corresponding to this dot.
+ */
+
+const defaultDot = (dotProps: DotProps): React.Element<any> => {
+    const {x, y} = dotProps;
+    return <circle fill='black' cx={x} cy={y} r={3}/>;
+};
+
+
+/**
+ *
+ * @component
+ *
+ * ScatterCanvas is the basic svg renderer for Scatter charts.
+ *
+ * @prop {number} [height]
+ * The width of the canvas. This is just passed on to the Canvas component.
+ *
+ * @prop {number} [width]
+ * The height of the canvas. This is just passed on to the Canvas component.
+ *
+ * @prop {Object} [svgProps]
+ * An object of props that will be spread onto the svg element. This is just passed on to the
+ * Canvas component.
+ *
+ * @prop {ChartData} data
+ * The `ChartData` Record used to contain the data for the chart.
+ *
+ * @prop {Scale} scaleX
+ * Any d3-scale for the x axis
+ *
+ * @prop {Scale} scaleY
+ * Any d3-scale for the y axis
+ *
+ * @prop {string} columnX
+ * The column key from `ChartData` to use for the x axis
+ *
+ * @prop {string} columnY
+ * The column key from `ChartData` to use for the y axis
+ *
+ * @prop {Dot} [dot]
```

Comment:
The description was a bit unclear but it doesn't actually default to a <circle/> element, it defaults to a wrapper component that then renders a circle. I've updated the wording to hopefully be a bit more clear.